### PR TITLE
palindrome

### PR DESCRIPTION
**Pull request by justintwed**
Palindrome exercise

### CODE REVIEW

**Comment by jay51** [comment]
good work

File: assignment3/justin/palindrome.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        if user_input[idx] != user_input[user_input_length - 1]:
+
+            not_palindrome(user_input)
+            return
+
+        user_input_length -= 1
+
+    is_palindrome(user_input)
+
+def main():
```
Comment (jay51) [inline assignment3/justin/palindrome.py:20]: add more comments

File: assignment3/justin/palindrome.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+def is_palindrome(user_input):
+    print(f"What you've entered {user_input} IS a palindrome.")
```
Comment (jay51) [inline assignment3/justin/palindrome.py:2]: why did you commit an empty  file

**Review by jay51 — CHANGES_REQUESTED** [review]

File: assignment3/justin/palindrome.py (hunk 1 above)
Comment (justintwed) [reply assignment3/justin/palindrome.py:20]: gotcha

**Pull request merged**